analyzer/tests/functional/ctu_failure/test_ctu_failure.py

```diff
@@ -135,7 +135,7 @@ def test_ctu_failure_zip(self):
         # with a specified name is analyzed.
         output, result = self.__do_ctu_all(
             on_demand=False, extra_args=["--verbose", "debug",
-                                         "-e", "debug.ExprInspection"])
+                                         "--saargs", "./ctu_failure.saargs"])
```

Comment:
Can you please update the documentation and highlight this information that debug checkers can be enabled only by using --saargs and not with the --enable option?

Also this is a backward compatibility change that if someone enabled debug checkers with the --enable option from this release these checkers will not be enabled. Please highlight this information too in the docs and also in the messag of this PR.

Reply:
Done in docs/analyzer/checker_and_analyzer_configuration.md